### PR TITLE
A small change in Documentation

### PR DESCRIPTION
**Pull request by rishibaghel25**
I have created this using the color used in brave website 
in this design i have given 3d effect and shadow
hope everyone will like 

if not then we can also use this 

Brave

### CODE REVIEW

**Review by shaharulislam999 — COMMENTED** [review]
>

**Comment by xemulat** [comment]
That is a very nice looking design!

**Comment by rishibaghel25** [comment]
....

File: README.md
Comment (1chooo) [inline README.md:1]: Your design looks great!

**Review by 1chooo — APPROVED** [review]

File: README.md
Comment (rishibaghel25) [reply README.md:1]: Thank you for liking it! I'm eagerly waiting for the merge.

**Comment by rishibaghel25** [comment]
Merge??🥺

**Review by Eason-S-Lu — APPROVED** [review]
Look nice.

**Comment by dipras** [comment]
It's nice
We need 1 more reviewer to merge it

**Comment by Eason-S-Lu** [comment]
> It's nice We need 1 more reviewer to merge it

Why are you not approving it though. (also we need code owner anyway)

**Review by dipras — APPROVED** [review]

**Comment by dipras** [comment]
> 

Done :D

**Comment by rishibaghel25** [comment]
merge :.. (

**Comment by bsclifton** [comment]
Thanks for the thought here - this logo is wrong. We have the correct assets here:
https://brave.com/media-assets/

Maybe @bradleyrichter or @jenn-rhim can help guide - anything with logo that folks will see needs a designer to review and approve

**Review by bsclifton — CHANGES_REQUESTED** [review]
See comment about wrong logo

We'll need approval from @bradleyrichter or @jenn-rhim before acceptance

**Comment by bradleyrichter** [comment]
Hello @rishibaghel25 - I think your graphic will work fine if you can replace the logo with the current version found at: https://brave.com/brave-branding-assets/

And also use Poppins for the typeface as shown on that page. Poppins Medium should work.

Thanks for the PR!

**Comment by rishibaghel25** [comment]
I have replaced the logo with the current version and also used Poppins.

File: README.md
Comment (bradleyrichter) [reply README.md:1]: that works! Thank you.

**Review by bradleyrichter — APPROVED** [review]

**Review by mihaiplesa — APPROVED** [review]

**Comment by Eason-S-Lu** [comment]
🎆

**Review by 1chooo — COMMENTED** [review]
That's nice!

**Pull request merged**
